source/faq/questions/dashboard-faq.md

```diff
@@ -55,6 +55,21 @@ When a run happens and a test fails - instead of going and inspecting your CI pr
 
 No, although we are looking to build an on-premise version of the Dashboard for use in private clouds. If you're interested in our on-premise version, please add your interest to our {% url "Dashboard Product Board" https://portal.productboard.com/cypress-io/1-cypress-dashboard %}!
 
+## {% fa fa-angle-right %} Why do my Cypress tests pass locally but not in CI?
+
+There are many reasons why tests may fail in CI but pass locally. Some of these include:
+
+- There is a problem isolated to the Electron browser (Tests run in CI use the Electron browser)
+- A test failure in CI could be highlighting a bug in your CI build process
+- Variability in timing when running your application in CI (For example, network requests that resolve within the timeout locally may take longer in CI)
+
```

Comment:
I might also add something about the machine differences in CI versus locally. You're running on a different machine so there may not be enough CPU, the environment settings may not be passed in properly, etc. Kind of like, machine setup issues.